src/sass/_profile_detail_overview.scss

```diff
@@ -51,6 +55,23 @@
     }
   }
 
+  .networks {
+    border-bottom: 1px solid $color-mid-light;
+
+    th:nth-child(1) {
+      width: 30%;
+    }
+
+    th:first-of-type,
+    td:first-of-type {
+      padding-left: 0;
+    }
+  }
```

Comment:
I think instead of duplicating the code from `_instance_detail_overview.scss`, better to extract them into a dedicated file -- as with the `_device_detail_list.scss`. We can have a `_network_detail_list.scss` as we introduced a reusable component for networks in the same way as we did previously for devices. So best to have the custom CSS also reused.

Also, I think the nth-child(1) and :first-of-type are the same, can we use one of the two and apply the two properties width and padding-left to that selector? That should simplify here. This is due to some legacy code, I think this simplification can be applied to other places in instance/profile overview as well, like in _device_detail_list and maybe other places as well.

Reply:
Hola!
As for that last paragraph, I don't think this can be condensed as nth child and first of type are different. In this case, nth child targets the first child of its parent, regardless of the type of element whilst first-of-type targets the first element of a specific type within its parent. Additionally, the latter is applied to table headers and table data cells and I don't think it would be useful to combine them.

Reply:
Ah, you are right!